frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,217 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+import RadioButton from 'primevue/radiobutton';
+import { Breadcrumb, Button } from '@/lib/primevue';
+import { RouteNames } from '@/utils/constants';
+import type { Ref } from 'vue';
+
+// State
+const enquiryOrIntake: Ref<string | null> = ref(null);
+const acceptDisclaimer: Ref<string> = ref('');
+
+// Actions
+const router = useRouter();
+const handleStartApplication = () => {
+  if (enquiryOrIntake.value === 'intakeform') {
+    router.push({ name: RouteNames.INTAKE });
+  } else if (enquiryOrIntake.value === 'enquiry') {
+    router.push({ name: RouteNames.ENQUIRY });
+  }
```

Comment:
Should be checking equivalence against the constants, not magic strings.
Can also be simplified (untested)
```ts
if ([RouteNames.ENQUIRY, RouteNames.INTAKE].includes(enquiryOrIntake.value)) {
    router.push({ name: enquiryOrIntake.value });
  }
```

Reply:
Excellent suggestion!
I had to do it slightly differently as this kept giving me the following error:
'Argument of type 'string' is not assignable to parameter of type '"enquiry" | "intakeform"'